docs/API-Publisher-Configuration.md

```diff
@@ -29,6 +29,7 @@ Defines general behavior of the Ed-Fi API Publisher.
 | Options:EnableRateLimit<br/>`--enableRateLimit`                                                           | Indicates whether or not to use rate limiting.<br/>(_Default value: false_) |
 | Options:RateLimitNumberExecutions<br/>`--rateLimitNumberExecutions`                                       | Indicates the maximum number of executions allowed within the defined time window.<br/>(_Default value: 100_) |
```

Comment:
Noticed that this documentation says default value is **100**. But in the app Settings file it changed to **30**.

Reply:
Fixed